siliconcompiler/toolscripts/ubuntu22/install-gtkwave.sh

```
@@ -5,19 +5,20 @@ set -ex
 # Get directory of script
 src_path=$(cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)/..
 
+# Install prerequisites only when they are missing
+. "${src_path}/_prereqs.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./_prereqs.sh was not specified as input (see shellcheck -x).

(SC1091)